### ISSUE TITLE
Popup ShouldDismissOnContentTouch = false not working

### ISSUE THREAD

**Issue by yusuphjoluwasen** [JonyFang/FFPopup#28]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](../CONTRIBUTING.md)
* [x] I have searched for a similar issue in the [project](https://github.com/JonyFang/FFPopup/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios
 Platform Version        | e.g. 11.0 / 12.1.2
 FFPopup Version         | e.g. 1.0.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 10.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by yusuphjoluwasen** [comment]
Popup still dismiss on content touch even after it has been set to true...any idea why this happens?
